CONTRIBUTING.md

```diff
@@ -2,18 +2,18 @@
 
 Thank you for the interest in Contributing to Clockify CLI.
 
-We accept pull requests for bug fixes and features (preferably that were discussed on a issue
+We accept pull requests for bug fixes and features (preferably that were discussed on an issue
 before). Also opening issues with feature requests and reporting bugs are very important
 contributions.
 
 Please do:
 
-- Check in the [issues][issues] if the [bug][bugs] or [feature request][features] has not been
-  submitted.
+- Check in the [issues](https://github.com/lucassabreu/clockify-cli/issues) if the [bug][bugs]
```

Comment:
add a `[issues]` referente to the end of the file instead
```md
[issues]: https://github.com/lucassabreu/clockify-cli/issues
```

Reply:
Done. I completely forgot about this way of linking.